src/metax_api/tests/models/catalog_record.py

```diff
@@ -97,7 +97,8 @@ def test_get_using_dict_with_id(self):
         self.assertEqual(obj.id, 1)
 
     def test_get_using_dict_with_urn_identifier(self):
-        row = {'research_dataset': {'urn_identifier': 'pid:urn:cr1'}, 'other_stuff': 'doesnt matter'}
+        row = {'research_dataset': {'urn_identifier': CatalogRecord.objects.first().research_dataset['urn_identifier']},
```

Comment:
Hint: There is a helper field in CR to access urn_identifier, so CatalogRecord.objects.first().urn_identifier works in the future too. Same for preferred_identifier.